app/views/devise/sessions/new.html.erb

```diff
@@ -15,8 +15,8 @@
             </div>
         <% end %>
         <% if session[:challenge_name].nil? %>
-            <div>
-                <%= link_to t('password.forgot_link'), forgot_password_path %>
+            <p class="govuk-body">
+                <%= link_to t('password.forgot_link'), forgot_password_path, class: 'govuk-link' %>
```

Comment:
Do we need `class: 'govuk-link'`? Doesn't this get added automatically as a result of using `govuk-frontend`?

Reply:
unfortunately not